docs/configuring-playbook-bot-draupnir.md

```diff
@@ -131,13 +143,13 @@ The first method intercepts the report API endpoint of the client-server API, wh
 matrix_bot_draupnir_abuse_reporting_enabled: true
 ```
 
-The other method polls an Synapse Admin API endpoint, hence it is available only if using Synapse and if the Draupnir user is an admin (see [above](#register-the-bot-account)). To enable it, set `pollReports: true` on `vars.yml` file as below.
+<!-- The other method polls an Synapse Admin API endpoint, hence it is available only if using Synapse and if the Draupnir user is an admin (see [above](#register-the-bot-account)). To enable it, set `pollReports: true` on `vars.yml` file as below. (This is unsupported by the playbook due to the admin API being inaccessible from containers currently.) -->
 
 ### Extending the configuration
 
 You can configure additional options by adding the `matrix_bot_draupnir_configuration_extension_yaml` variable.
 
-For example, to change Draupnir's `pollReports` option to `true`, add the following configuration to your `vars.yml` file:
+For example, to change Draupnir's `acceptInvitesFromSpace` option to `!example:example.org`, add the following configuration to your `vars.yml` file:
```

Comment:
```suggestion
For example, to change Draupnir's `acceptInvitesFromSpace` option to `!qporfwt:example.com`, add the following configuration to your `vars.yml` file:
```

Reply:
As room IDs are literally free to be whatever you want i dont see the purpose of this example value change. The TLD that might make sense. Readable room IDs with clear meanings are common in communities where users run modified Synapse that allows this.

Reply:
Please check https://github.com/spantaleev/matrix-docker-ansible-deploy/issues/3654 for the context.